### PR TITLE
TPD-441 Pushpay integration fix

### PR DESCRIPTION
**Pull request by dahnbaez**
Changes in the way we redirect back to touchpoint after successful payment.
https://touchpointsoftware.atlassian.net/secure/RapidBoard.jspa?rapidView=5&projectKey=TPD&assignee=5e19301c85a8c90ecaca7d44

### CODE REVIEW

File: CmsWeb/Areas/Setup/Controllers/PushpayController.cs [hunk 1]
```diff
@@ -223,44 +260,66 @@ public async Task<ActionResult> Registration(int DatumId)
...
-                if (sr.Substring(0, 9) == "payamtdue")
-                {
-                    Payment payment = await _pushpayPayment.GetPayment(paymentToken);
-                    return PayAmtDueProcess(payment, Int32.Parse(sr.Substring(10)));
+                    case "pd":
+                        return PayAmtDueProcess(payment, Int32.Parse(reference.Substring(2)));
                 }
-
-                throw new Exception("Registration Type is not supported for Pushpay");
+                //if (sr.Substring(0, 3) == "Org")
```
Comment (josearmandohf) [inline CmsWeb/Areas/Setup/Controllers/PushpayController.cs]: Code commented from line 300 to 321. If is not used anymore please remove it.

File: CmsWeb/Web.config [hunk 2]
```diff
@@ -59,6 +59,7 @@
     <add key="OrgBaseDomain" value="tpsdb.com" />
     <add key="TouchpointAuthServer" value="https://123ec8c6.ngrok.io/pushpay/complete" />
     <add key="IsDeveloperMode" value="false" />
+    <add key="PushpayRU" value="touchpointest" />
```
Comment (josearmandohf) [inline CmsWeb/Web.config]: Make sure the variable is changed in production for the real value.

**Comment by josearmandohf** [comment]
I have updated the branch and leave some simple comments to address before approval.

File: CmsWeb/Web.config (hunk 2 above)
Comment (brondavies) [reply CmsWeb/Web.config]: I've taken care of this - thanks @josearmandohf

**Pull request merged**